driver-sync/src/test/functional/com/mongodb/client/ClientSideEncryptionExternalKeyVaultTest.java

```diff
@@ -73,16 +69,10 @@ public void setUp() throws IOException, URISyntaxException {
                 System.getProperty("org.mongodb.test.awsAccessKeyId") != null
                     && !System.getProperty("org.mongodb.test.awsAccessKeyId").isEmpty());
 
-        CodecRegistry codecRegistry = fromRegistries(
-                fromCodecs(new UuidCodec(UuidRepresentation.STANDARD)),
-                MongoClientSettings.getDefaultCodecRegistry());
         MongoClientSettings.Builder clientSettingsBuilder = getMongoClientSettingsBuilder();
```

Comment:
Move closer to first use

Reply:
I replaced it on `getMongoClientSettingsBuilder` as suggested in the `async` test